Index: linux/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts
===================================================================
--- linux.orig/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts
+++ linux/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts
@@ -662,16 +662,13 @@
 
 &vop {
 	status = "okay";
+    disable-win-move;
 };
 
 &vop_mmu {
 	status = "okay";
 };
 
-&vepu {
-	status = "okay";
-};
-
 /* vp0 & vp1 splice for 8K output */
 &vp0 {
 	rockchip,plane-mask = <(1 << ROCKCHIP_VOP2_CLUSTER0 | 1 << ROCKCHIP_VOP2_ESMART0)>;
