Index: linux/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts
===================================================================
--- linux.orig/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts
+++ linux/arch/arm64/boot/dts/rockchip/rk3588-rock-5b.dts
@@ -578,6 +578,7 @@
 
 &hdptxphy_hdmi_clk0 {
 	status = "okay";
+    cec-enable = "true";
 };
 
 &hdptxphy_hdmi_clk1 {
@@ -586,7 +587,7 @@
 
 &hdmi0 {
 	status = "okay";
-        cec-enable = "true";
+	cec-enable = "true";
 };
 
 &hdmi0_in_vp0 {
@@ -609,7 +610,7 @@
 	status = "okay";
 	pinctrl-names = "default";
 	pinctrl-0 = <&hdmim0_tx1_cec &hdmim0_tx1_hpd &hdmim1_tx1_scl &hdmim1_tx1_sda>;
-        cec-enable = "true";
+	cec-enable = "true";
 };
 
 &hdmi1_in_vp0 {
