Index: linux/drivers/net/wireless/rockchip_wlan/rtl8852be/Makefile
===================================================================
--- linux.orig/drivers/net/wireless/rockchip_wlan/rtl8852be/Makefile
+++ linux/drivers/net/wireless/rockchip_wlan/rtl8852be/Makefile
@@ -192,6 +192,7 @@ ifeq ($(CONFIG_HWSIM), y)
 else
 	HAL = phl
 endif
+DRV_PATH = $(S)/drivers/net/wireless/rockchip_wlan/rtl8852be
 
 ifeq ($(CONFIG_PLATFORM_RTL8198D), y)
 DRV_PATH = $(src)
Index: linux/drivers/net/wireless/rockchip_wlan/rtl8852bu/Makefile
===================================================================
--- linux.orig/drivers/net/wireless/rockchip_wlan/rtl8852bu/Makefile
+++ linux/drivers/net/wireless/rockchip_wlan/rtl8852bu/Makefile
@@ -201,6 +201,8 @@ endif
 
 DRV_PATH = $(S)/drivers/net/wireless/rockchip_wlan/rtl8852bu
 
+DRV_PATH = $(S)/drivers/net/wireless/rockchip_wlan/rtl8852bu
+
 ########### HAL_RTL8852A #################################
 ifeq ($(CONFIG_RTL8852A), y)
 IC_NAME := rtl8852a
